typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.stories.tsx

```diff
@@ -415,6 +454,122 @@ AllWellHeadsHidden.parameters = {
     },
 };
 
+const testWell: FeatureCollection = {
+    type: "FeatureCollection",
+    features: [
+        {
+            type: "Feature",
+            geometry: {
+                type: "GeometryCollection",
+                geometries: [
+                    {
+                        type: "Point",
+                        coordinates: [0, 0],
+                    },
+                    {
+                        type: "LineString",
+                        coordinates: [
+                            [0, 0, 0],
+                            [0, 0, 1],
+                            [0, 0, 2],
+                            [0, 50, -50],
+                            [0, 0, -100],
+                            [99, 99, -150],
+                            [99, 0, -250],
+                        ],
+                    },
+                ],
+            },
+            properties: {
+                name: "well99",
+                color: [255, 255, 0, 255],
+                md: [[0, 1, 2, 3, 4, 5, 8, 9]],
+            },
+        },
+    ],
+};
+
+const BBox = [-100, -100, -250, 100, 100, 0] as [
+    number,
+    number,
+    number,
+    number,
+    number,
+    number,
+];
+
+export const WellsRefine: StoryFn<typeof SubsurfaceViewer> = (args) => {
+    const [refineNumber, setRefineNumber] = React.useState<number>(1);
+
+    const props = {
+        ...args,
+        layers: [
+            {
+                "@@type": "WellsLayer",
```

Comment:
Should use the class syntax in order to get typing

Reply:
For some reason it doe not work witn "new". It does not pick up the input data changes. I suggest maybe we could do it this way fo rnow and later investigate further why "new" does not always work,